### ISSUE TITLE
makePath fails when it should be inheriting route parameters

### ISSUE THREAD

**Issue by TehShrike** [TehShrike/abstract-state-router#68]
Failing test committed in https://github.com/TehShrike/abstract-state-router/commit/9739c3949dcef8fb84d82eedeac8aaa6ec9898a9

Fails with an error:
```
/Users/josh/code/abstract-state-router/index.js:279
							throw e
							^

Error: Must supply argument someParam for path /parent/:someParam/yarp/child1
    at /Users/josh/code/abstract-state-router/node_modules/page-path-builder/index.js:17:11
```

Reported by @daytonlowell

**Comment by TehShrike** [comment]
Found and fixed.  Another followup question I'll think about eventually maybe: should `stateIsActive` start returning true as soon as the state transition starts, or should it not return true until after `stateChangeEnd`, like it does now?

**Comment by TehShrike** [comment]
Published as 5.12.3
